src/screens/Ticketing/Purchase/Travellers/index.tsx

```diff
@@ -210,40 +226,47 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
     padding: theme.spacings.medium,
     marginTop: theme.spacings.small,
   },
-  travellerContainer: {
-    backgroundColor: theme.background.level0,
-    borderBottomEndRadius: theme.border.radius.regular,
-    borderBottomLeftRadius: theme.border.radius.regular,
-    padding: theme.spacings.medium,
-    marginBottom: theme.spacings.small,
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  travellerCount: {flex: 1, flexDirection: 'row', alignItems: 'center'},
-  travellerCountActions: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-    width: 100,
+  header: {
+    paddingHorizontal: theme.spacings.medium,
+    marginBottom: theme.spacings.medium,
   },
   totalContainer: {
     flexDirection: 'row',
     justifyContent: 'space-between',
     padding: theme.spacings.medium,
+    backgroundColor: theme.background.level0,
+    borderRadius: theme.border.radius.regular,
+  },
+  totalContainerHeadings: {
+    flexDirection: 'column',
+    paddingVertical: theme.spacings.xSmall,
   },
-  informationLink: {
-    textAlign: 'center',
+  footer: {
+    paddingTop: theme.spacings.medium,
     paddingHorizontal: theme.spacings.medium,
-    paddingVertical: theme.spacings.small,
+    backgroundColor: theme.background.header,
   },
   buttons: {
+    flexDirection: 'row',
+  },
+  paymentButton: {
+    marginTop: theme.spacings.medium,
+    flex: 1,
+    minHeight: 64,
+  },
+  vippsPaymentButton: {
+    marginRight: 6,
+  },
+  cardPaymentButton: {
+    marginLeft: 6,
+  },
```

Comment:
I think we have to look into the use of these styles.  My thinking is that flexbox could probably solve some of these problems for us. Also should rather use `theme.spacings` here rather than explicit numbers.